resource/StandardResources.md

```diff
@@ -0,0 +1,40 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus.
+
+## Kubernetes (k8s)
+
+Resources specific to Kubernetes:
+
+### `k8s.io/container`
+**Description:** A Kubernetes container instance.
+
+**Labels:**
+* `k8s.io/cluster/name`: The name of the cluster that the container is running in.
+* `k8s.io/namespace/name`: The name of the namespace that the container is running in.
+* `k8s.io/pod/name`: The name of the pod that the container is running in.
+* `k8s.io/container/name`: The name of the container.
```

Comment:
Do we want `/` style names? In attributes we are using dot style like `http.method`. Do we need consistency here?

Reply:
What is your suggestion? Happy with both

Reply:
I was mostly wondering if these names are aligned with some other naming convention I'm not aware of. If this is a new art I'd suggest to use consistent notation accross attribute and resource attributes. So I'd advice for dot-separated names

Reply:
Dot-separated names we have now :)

Reply:
Also, why k8s.io and not **kubernetes**?